src/app/persistence/AttributePersistence.h

```diff
@@ -53,19 +53,27 @@ class AttributePersistence
     ///   - decode the given raw data
     ///   - write to storage
     template <typename T, typename std::enable_if<std::is_arithmetic_v<T>>::type * = nullptr>
```

Comment:
```suggestion
    template <typename T, typename std::enable_if_t<std::is_arithmetic_v<T>> * = nullptr>
```
Like the other place (since I have some fixes to comments anyway).